src/deployment-guides/tko-saas-services.md

```diff
@@ -287,6 +287,14 @@ Note:
 
    ![Namespace Exclusion](img/tko-saas-services/tko-deploy-saas20.jpg)
 
+   - If using user managed packages such as cert-manager, contour, prometheus, and grafana, you will want to exclude those namespaces.
```

Comment:
If you install user-managed packages such as Cert Manager, Contour, Prometheus, and Grafana, exclude the related namespaces for the user-managed packages: